server/src/test/scala/com/github/saxypandabear/songrequests/websocket/TwitchSocketSpec.scala

```diff
@@ -41,15 +41,18 @@ class TwitchSocketSpec extends UnitSpec {
     socket.inputMatchesSpotifyUri("") should be(false)
   }
 
-  "Checking a reward title that contains the phrase 'song request'" should "return true" in {
-    socket.isSongRequest("This is a song request reward") should be(true)
-    socket.isSongRequest("SONG REQUEST ALL CAPS") should be(true)
+  "Checking a reward with the title 'TwitchSongRequests'" should "return true" in {
+    socket.isSongRequest("This is a TwitchSongRequests reward") should be(true)
+    socket.isSongRequest("TwitchSongRequests") should be(true)
   }
 
   "Checking a reward title that does not contain the exact phrase" should "return false" in {
     socket.isSongRequest("This has a song, but is not a request") should be(
         false
     )
+    socket.isSongRequest("twitchsongrequests") should be(false)
+    socket.isSongRequest("This is a song request reward") should be(true)
```

Comment:
These tests are copy pasted wrong, and also don't actually test the thing it should be